File: HealthSysHub.Web.Utility/Models/PharmacyOrderItemDetails.cs
namespace HealthSysHub.Web.Utility.Models
{
    public class PharmacyOrderItemDetails
    {
        public Guid PharmacyOrderItemId { get; set; }
        public Guid? PharmacyId { get; set; }
        public Guid? MedicineId { get; set; }
        public string? MedicineName { get; set; }
        public long? ItemQty { get; set; }
        public decimal? UnitPrice { get; set; }
        public decimal? TotalAmount { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTimeOffset? CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTimeOffset? ModifiedOn { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/pharmacy-order-item-details.ts
export interface PharmacyOrderItemDetails {
  pharmacyOrderItemId: string; // Guid as string
  pharmacyId?: string | null; // Guid as string
  medicineId?: string | null; // Guid as string
  medicineName?: string | null;
  itemQty?: number | null; // long as number
  unitPrice?: number | null; // decimal as number
  totalAmount?: number | null; // decimal as number
  createdBy?: string | null; // Guid as string
  createdOn?: string | null; // DateTimeOffset as ISO string
  modifiedBy?: string | null; // Guid as string
  modifiedOn?: string | null; // DateTimeOffset as ISO string
  isActive: boolean;
}
